### ISSUE TITLE
ev3dev codegen: global statements not generated for functions

### ISSUE THREAD

**Issue by drigz** [OpenRoberta/robertalab#318]
Steps to reproduce:
1. Create a program with one variable and one function:
![image](https://cloud.githubusercontent.com/assets/632207/20828708/11509526-b879-11e6-8e48-573c217d086e.png)
1. Connect to an ev3dev robot.
1. Go edit -> show code.

No global statement is generated for item in doSomething():

```
item = 0
def run():
    global item
    
def doSomething():
 
def main():
    ...
```

**Comment by ensonic** [comment]
Somewhat related, #292

**Comment by bjost2s** [comment]
will be fixed in #292
